source/help/messaging/flagging-messages.rst

```diff
@@ -1,7 +1,7 @@
 Flagging Messages
 -------------------
 
-Flags are a way to mark messages for follow up. Your flags are personal, and cannot be seen by other users.
+Flags are a way to mark messages for follow up. Your flags are personal, and cannot be seen by other users. There is no limit to the number of flagged posts in a channel.
```

Comment:
Sorry for the nits. Flagged posts don't live in a channel so this doesn't make sense. We should say "There is no limit to the number of posts you can flag."